apps/tx-builder/src/components/CreateNewBatchCard.tsx

```diff
@@ -40,65 +44,91 @@ const CreateNewBatchCard = ({ onFileSelected }: CreateNewBatchCardProps) => {
   return (
     <Wrapper isSmallScreen={isSmallScreen}>
       <Hidden smDown>
-        <CreateNewBatchSVG />
+        {mode === EModes.DARK ? <CreateNewBatchDarkSvg /> : <CreateNewBatchLightSvg />}
+        <StyledArrowBlock />
       </Hidden>
-      <StyledDragAndDropFileContainer
-        {...dropHandlers}
-        dragOver={isOverDropZone}
-        fullWidth={isSmallScreen}
-        error={isAcceptError}
-      >
-        {isAcceptError ? (
-          <StyledText variant="body1" error={isAcceptError}>
-            The uploaded file is not a valid JSON file
-          </StyledText>
-        ) : (
-          <>
-            <Icon type="termsOfUse" size="sm" />
-            <StyledText variant="body1">Drag and drop a JSON file or</StyledText>
-            <StyledButtonLink color="secondary" onClick={handleBrowse}>
-              choose a file
-            </StyledButtonLink>
-          </>
-        )}
-      </StyledDragAndDropFileContainer>
-      <input
-        ref={fileRef}
-        id="logo-input"
-        type="file"
-        onChange={handleFileSelected}
-        accept=".json"
-        hidden
-      />
+
+      <StyledCreateBatchContent>
+        <StyledText variant="body1">Start creating a new batch </StyledText>
+        <StyledText variant="body1">or</StyledText>
+        <StyledDragAndDropFileContainer
+          {...dropHandlers}
+          dragOver={isOverDropZone}
+          fullWidth={isSmallScreen}
+          error={isAcceptError}
+        >
+          {isAcceptError ? (
+            <StyledText variant="body1" error={isAcceptError}>
+              The uploaded file is not a valid JSON file
+            </StyledText>
+          ) : (
+            <>
+              <Icon type="termsOfUse" size="sm" />
+              <StyledText variant="body1">Drag and drop a JSON file or</StyledText>
+              <StyledButtonLink color="secondary" onClick={handleBrowse}>
+                choose a file
+              </StyledButtonLink>
```

Comment:
From the designs it looks like this field should be white and only turn green when dragging a file over it. And the error when dragging an unsupported file should have a different shade of red:

Current:
<img width="457" alt="Screenshot 2024-10-18 at 10 35 48" src="https://github.com/user-attachments/assets/a01adec7-4b53-489b-ba46-d88bd975a301">

Design:
<img width="640" alt="Screenshot 2024-10-18 at 10 33 40" src="https://github.com/user-attachments/assets/83328127-1651-4ba5-8c25-5941789c9f13">
<img width="515" alt="Screenshot 2024-10-18 at 10 36 13" src="https://github.com/user-attachments/assets/33df7752-f640-4de7-9939-e535bdd13802">


Current:
<img width="466" alt="Screenshot 2024-10-18 at 10 34 59" src="https://github.com/user-attachments/assets/b77fe44d-1439-4f41-a942-ad6cccc47502">

Design:
<img width="422" alt="Screenshot 2024-10-18 at 10 35 33" src="https://github.com/user-attachments/assets/09edf281-4066-4cf4-8ebc-0cad18c2b612">

Reply:
Hey @usame-algan we are using the same component and colors from the web-wallet. That is why I didn't change this one.